cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -783,3 +793,245 @@ def test_search_disabled(self, store_type):
     @ddt.data(*WORKS_WITH_STORES)
     def test_exception(self, store_type):
         self._perform_test_using_store(store_type, self._test_exception)
+
+
+class GroupConfigurationSearchMongo(CourseTestCase, MixedWithOptionsTestCase):
```

Comment:
I'm also inheriting CourseTestCase, which inherits ModuleStoreTestCase, which has store switch built-in using this approach. It looked simpler to me and requires less test code to be store-aware, but I can switch to ddt if that is what you prefer.